types/externs.ts

```diff
@@ -4,7 +4,7 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-import {Artifacts} from './artifacts';
```

Comment:
This file is kinda weird. It houses some necessary type definitions, but also modifies the global namespace. I propose we remove this file, move the type definitions to a `*.ts` file and the global namespace stuff to `node.d.ts` or something.

Reply:
#14537